### PR TITLE
Article heading

### PR DESCRIPTION
**Pull request by Danwhy**
Builds on #10 
Adds article heading to the wagtail model, with some help text to describe what the fields are.

### CODE REVIEW

File: articles/models.py
Comment (samhstn) [inline articles/models.py:32]: [Nice](https://docs.djangoproject.com/en/1.11/topics/db/models/)

**Review by samhstn — COMMENTED** [review]
Code looks good 👍 

@Danwhy Have you got this running with Heroku yet? (I'll check out what the meta stuff has done from there if so)

**Comment by samhstn** [comment]
After #10 is merged we can rebase and let @nelsonic review

**Comment by Danwhy** [comment]
@shouston3 I haven't pushed it to heroku yet, would you like me to?

**Comment by samhstn** [comment]
@Danwhy Heroku looks down atm, with error:

```bash
column home_homepage.footer does not exist
LINE 1: ..._homepage"."page_ptr_id", "home_homepage"."body", "home_home...
```

Will merge once the heroku database has received this new migration and is all working

**Comment by samhstn** [comment]
@nelsonic This is rebased and now all good to go (check out [heroku](ldmw-cms.herokuapp.com/admin))

**Review by nelsonic — APPROVED** [review]
@Danwhy looks great. 👀  👍 
@shouston3 thanks for peer-reviewing. ✅

**Pull request merged**
